crates/astria-bridge-withdrawer/src/bridge_withdrawer/startup.rs

```diff
@@ -409,12 +409,11 @@ async fn wait_for_empty_mempool(
                 futures::future::ready(())
             },
         );
-    let sequencer_client =
-        SequencerServiceClient::connect(format!("http://{}", sequencer_grpc_endpoint.clone()))
-            .await
-            .wrap_err_with(|| {
-                format!("failed to connect to sequencer at `{sequencer_grpc_endpoint}`")
-            })?;
+    let sequencer_client = SequencerServiceClient::connect(sequencer_grpc_endpoint.clone())
```

Comment:
We shouldn't be forcing http on the url path here, uri protocol specified in the env var.